client/src/assets/index.js

```diff
@@ -13,6 +13,9 @@ import EVENT_11 from './events-page/past-events/22_23/11.svg';
 import EVENT_12 from './events-page/past-events/22_23/12.svg';
 import EVENT_13 from './events-page/past-events/22_23/13.svg';
 import EVENT_14 from './events-page/past-events/22_23/14.svg';
+import EVENT_15 from './events-page/past-events/23_24/1.svg';
+
```

Comment:
And since this file contains all past events, this file should be located inside the `past-events` folder. 

Note: Don't hesitate to change things if you think it's right, it'll get reviewed anyway.

Reply:
There is no current index.js inside past-events folder, am I looking at the right one?

Reply:
I think it's an error on my local. Anyways, yes just move this file into the `past-events` folder.